front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -6,12 +6,15 @@ import { isEmpty, noop } from 'lodash';
 import { FetchBaseQueryError } from '@reduxjs/toolkit/query/react';
 
 import { setFailure } from 'reducers/main';
-import { updateSpeedLimitByTag } from 'reducers/osrdconf';
-import { getInfraID, getSpeedLimitByTag } from 'reducers/osrdconf/selectors';
 import SelectImprovedSNCF from 'common/BootstrapSNCF/SelectImprovedSNCF';
 import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
 import icon from 'assets/pictures/components/speedometer.svg';
 import './SpeedLimitByTagSelector.scss';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
+import { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
````suggestion
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
import type { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Reply:
Done